test/Microsoft.AspNet.Mvc.FunctionalTests/FlushPointTest.cs

```diff
@@ -92,112 +60,44 @@ public class FlushPointTest
         public async Task FlushPointsAreExecutedForPagesWithComponentsPartialsAndSections(string action, string title)
         {
-            var waitService = new WaitService();
-            var server = TestHelper.CreateServer(_app, SiteName,
-                services =>
-                {
-                    _configureServices(services);
-                    services.AddInstance(waitService);
-                });
-            var client = server.CreateClient();
+            var expected = $@"<title>{ title }</title>
+RenderBody content
 
-            // Act
-            var stream = await client.GetStreamAsync("http://localhost/FlushPoint/" + action);
-
-            // Assert - 1
-            Assert.Equal(
-                $@"<title>{ title }</title>
-RenderBody content",
-                GetTrimmedString(stream),
-                ignoreLineEndingDifferences: true);
-            waitService.WaitForServer();
 
-            // Assert - 2
-            Assert.Equal(
-                @"partial-content
+partial-content
 
 Value from TaskReturningString
-<p>section-content</p>",
-                GetTrimmedString(stream),
-                ignoreLineEndingDifferences: true);
-            waitService.WaitForServer();
-
-            // Assert - 3
-            Assert.Equal(
-                @"component-content
+<p>section-content</p>
+    component-content
     <span>Content that takes time to produce</span>
 
-More content from layout",
-                GetTrimmedString(stream),
-                ignoreLineEndingDifferences: true);
+More content from layout
+";
+
+            // Act
+            var body = await Client.GetStringAsync("http://localhost/FlushPoint/" + action);
+
+            // Assert
+            Assert.Equal(expected, body, ignoreLineEndingDifferences: true);
         }
 
         [Fact]
         public async Task FlushPointsNestedLayout()
         {
             // Arrange
-            var waitService = new WaitService();
-            var server = TestHelper.CreateServer(_app, SiteName,
-                services =>
-                {
-                    _configureServices(services);
-                    services.AddInstance(waitService);
-                });
-            var client = server.CreateClient();
+            var expected = @"Inside Nested Layout
+<title>Nested Page With Layout</title>
 
-            // Act
-            var stream = await client.GetStreamAsync("http://localhost/FlushPoint/PageWithNestedLayout");
-
-            // Assert - 1
-            Assert.Equal(
-                @"Inside Nested Layout
-<title>Nested Page With Layout</title>",
-                GetTrimmedString(stream),
-                ignoreLineEndingDifferences: true);
-            waitService.WaitForServer();
-
-            // Assert - 2
-            Assert.Equal("<span>Nested content that takes time to produce</span>", GetTrimmedString(stream));
-        }
 
-        [Fact]
-        public async Task FlushBeforeCallingLayout()
```

Comment:
That's why I removed it entirely and mentioned the unit test in the commit description.